src/Workspaces/Core/Portable/Options/EditorConfigStorageLocation.cs

```diff
@@ -49,11 +53,19 @@ public EditorConfigStorageLocation(string keyName)
             {
                 if (type == typeof(int))
                 {
-                    return int.Parse(s);
+                    if (int.TryParse(s, out var value))
+                    {
+                        return value;
+                    }
+                    return null;
```

Comment:
null means we couldn't parse it so return the default value.

---
In reply to: [102832529](https://github.com/dotnet/roslyn/pull/17097#discussion_r102832529) [](ancestors = 102832529)